packages/gestalt/src/PageHeader.css

```diff
@@ -0,0 +1,9 @@
+.pageHeader {
```

Comment:
For this, I'd recommend using
border-bottom: 1px solid var(--color-border-container)

Then you wont need separate CSS for dark mode

Reply:
DONE